special-pages/index.mjs

```diff
@@ -156,6 +159,10 @@ for (const buildJob of buildJobs) {
             },
             dropLabels: buildJob.injectName === 'integration' ? [] : ['$INTEGRATION'],
         });
+        if (output.metafile) {
+            const meta = JSON.stringify(output.metafile, null, 2);
+            writeFileSync(join(buildJob.outputDir, 'metafile.json'), meta);
+        }
```

Comment:
this produces an output file you can drop into https://esbuild.github.io/analyze/ - it was useful for me debugging why the auto-animate library was being pulled into the NTP